### PR TITLE
Refactoring ~historical price timeseries~ risk contributions calculations

### PR DESCRIPTION
**Pull request by acemasterjb**
This PR is changed from refactoring the historical price timeseries to refactoring the risk contributions calculations, mainly to verify calculations.

### CODE REVIEW

File: scheduler/app/libs/tasks/__init__.py [hunk 1]
```diff
@@ -9,3 +9,4 @@
 from ..pd_inter_calc import portfolio_midnight_filler
 
 from ..types import Treasury
+from .get_assets import *
```
Comment (lajarre) [inline scheduler/app/libs/tasks/__init__.py:12]: Let's try to always import explicitely: https://stackoverflow.com/questions/2386714/why-is-import-bad

File: scheduler/app/libs/tasks/get_assets.py [hunk 2]
```diff
@@ -126,12 +135,28 @@ def augment_total_balance(
...
 
 async def build_treasury_with_assets(
-    treasury_address: str, chain_id: int, start: str, end: str
+    treasury_address: str, chain_id: int, start: str = None, end: str = None
 ) -> tuple[Treasury, dict[str, DF], dict[str, DF]]:
     treasury = filter_out_small_assets(await make_treasury(treasury_address, chain_id))
     augmented_token_hist_prices = augment_token_hist_prices(
         await get_token_hist_prices(treasury)
     )
+
```
Comment (lajarre) [inline scheduler/app/libs/tasks/get_assets.py:144]: I think we should not add any more code in this function, as it's a very central point of failure. Also, having default values for `start` and `end` is useful sometimes, but not always. For instance, `main.portfolio` endpoint doesn't need defaults.

Just add your date-making logic inside of `reload_treasuries_data`.

File: scheduler/app/libs/tasks/get_assets.py [hunk 3]
```diff
@@ -126,12 +135,28 @@ def augment_total_balance(
...
+    if not start:
+        from datetime import datetime
```
Comment (lajarre) [inline scheduler/app/libs/tasks/get_assets.py:146]: In terms of the imports, let's keep them always at start of the file. When we'll introduce a linter, this code will not pass. Also, you don't need mktime here.

**Review by lajarre — CHANGES_REQUESTED** [review]

File: scheduler/app/libs/tasks/get_assets.py (hunk 2 above)
Comment (acemasterjb) [reply scheduler/app/libs/tasks/get_assets.py:147]: noted, changes have been made

**Pull request merged**